dashboard/config/scripts/levels/2019 Dance Lab Project.level

```diff
@@ -59,7 +59,7 @@
   <blocks>
     <start_blocks>
       <xml>
-        <block type="Dancelab_whenSetup" deletable="false">
+        <block type="Dancelab_whenSetup" deletable="false" movable="false">
```

Comment:
meta-question: is there no source of truth for this button? If you create a new dance party level what does the default `whenSetup` button look like?

Reply:
Great question! I've only ever cloned existing levels, not created new ones. I just checked and looks like the default start blocks for a new level is completely blank - no `whenSetup` block at all! This means a levelbuilder would need to add one in start mode and then toggle on each of these attributes.

Reply:
If we wanted to add a source of truth, we could do that by adding a `start_blocks` prop here: https://github.com/code-dot-org/code-dot-org/blob/7d8cb181e56f0ee28e563a4d42673df9f64ee91d/dashboard/app/models/levels/dancelab.rb#L38
This could be a pretty light follow-up PR that we'd probably want to merge before anyone makes more Dance levels. (No current plans for that AFAIK)

Reply:
Add a Jira ticket to this PR's follow-up section: https://codedotorg.atlassian.net/browse/SL-524